webapp/src/main/webapp/src/assets/template/groups-template.csv

```diff
@@ -0,0 +1 @@
+group_name,school_natural_id,school_year,subject_code,student_ssid,group_user_login

```

Comment:
Thought about that. Or even giving them a separate "Sample" download beside the "Template" download. As a user I think some hard examples would be useful.